### ISSUE TITLE
Nested element node .setChildren not working

### ISSUE THREAD

**Issue by grahnj** [ballerina-platform/ballerina-lang#20609]
**Description:**
In scenarios where an element node is nested in an XML document, the setChildren method fails, despite the .getItemType() method returning element. As such, there is no way in Ballerina 1.1.0 to set the children/text value of the afformentioned node.

**Steps to reproduce:**

Executing the main function below will result in the setChildren method panicking on the line "nested.setChildren("New value");

```
public function main() {
    xml elementNode = xml`<elementNode>Old value</elementNode>`;
    elementNode.setChildren("New value"); 

    io:println("working", elementNode);

    xml elementNestedNode = xml`<elementNode><nested>Old value</nested></elementNode>`;
    xml nested = elementNestedNode["nested"];

    
    io:println(nested.getItemType()); // element
    
    nested.setChildren("New value"); // panicks

    io:println("not working", elementNestedNode); // never called
}
```

Using getItemType(), it is verified that the node in question is in fact of type ELEMENT. 

**Affected Versions:**

1.1.0

**Suggested Labels (optional):**
xml, xml functions

**Comment by rdhananjaya** [comment]
Same issue seems to be there ins x.getChildren() function as well.
The root cause for this is a bug in xml-langlib.isElement() function.

**Comment by rdhananjaya** [comment]
Fixed in 1.1.2
